### ISSUE TITLE
Error when evaluating SSR module /src/hooks.server.ts

### ISSUE THREAD

**Issue by WarningImHack3r** [inlang/inlang#699]
## Problem

```
[vite] Error when evaluating SSR module /src/hooks.server.ts: failed to import "/node_modules/.pnpm/@inlang+sdk-js@0.3.1/node_modules/@inlang/sdk-js/dist/adapter-sveltekit/runtime/server/index.js"
```
My `hooks.server.ts` file has been generated by inlang and only contains the comment explaining it, nothing else.

### Expected behavior

To work

## Steps to reproduce

1. Install the latest version on a SvelteKit project with the following `inlang.config.js`:
```javascript
export async function defineConfig(env) {
	const { default: pluginJson } = await env.$import(
		"https://cdn.jsdelivr.net/gh/samuelstroschein/inlang-plugin-json@2/dist/index.js"
	);

	const { default: sdkPlugin } = await env.$import(
		"https://cdn.jsdelivr.net/npm/@inlang/sdk-js@0.3.1/dist/plugin/index.js"
	);

	return {
		referenceLanguage: "en",
		plugins: [
			pluginJson({
				pathPattern: "./src/locales/{language}.json"
			}),
			sdkPlugin({
				languageNegotiation: {
					strategies: [{ type: "localStorage" }]
				}
			})
		]
	};
}
```
and Vite config:
```javascript
import inlangPlugin from "@inlang/sdk-js/adapter-sveltekit"
import { sveltekit } from "@sveltejs/kit/vite";
import { defineConfig } from "vite";

export default defineConfig({
	plugins: [inlangPlugin(), sveltekit()],
    // I guess everything below is optional but this is what I have
	server: {
		fs: {
			allow: ["."]
		}
	}
});
```
2. `npm run dev`

**Comment by WarningImHack3r** [comment]
@inlang any update?

**Comment by ivanhofer** [comment]
@WarningImHack3r can you please check if this issue is resolved in version `0.6.0` of the sdk?
If not please create a repository where we can reproduce this issue.

**Comment by WarningImHack3r** [comment]
It indeed solves the crash! Thank you!
